app/src/features/datasets/components/DatasetArtifacts.tsx

```diff
@@ -198,6 +216,7 @@ const DatasetAttachments: React.FC<IDatasetAttachmentsProps> = (props) => {
     <>
       <ActionToolbar label="Documents" labelProps={{ variant: 'h4' }}>
         <Box display="flex" gap={1}>
+          {/*
```

Comment:
do we need to keep this commented out code?

Reply:
Good question! @jeznorth Jer asked me to leave it in.